js/ad_manager_vast.js

```diff
@@ -1788,6 +2097,9 @@ OO.Ads.manager(function(_, $) {
         result.maintainAspectRatio = nonLinear.attr("maintainAspectRatio");
         result.minSuggestedDuration = nonLinear.attr("minSuggestedDuration");
         result.nonLinearClickThrough = nonLinear.find("NonLinearClickThrough").text();
+        result.nonLinearClickTracking = filterEmpty($(nonLinearAdsXml).
```

Comment:
Should we be checking to see if the filtered results is empty and if the find returns nothing?